### ISSUE TITLE
Missing flute.wav

### ISSUE THREAD

**Issue by sibble-dev** [pincoide/UO-EC-Super-Viewer#1]
Invalid Resx file. Could not find a part of the path '..\bin\Debug\Test Spritesheets\flute.wav'. Line 10112, position 5.	UO EC Super Viewer	..\UO-EC-Super-Viewer\Properties\Resources.resx

**Comment by sibble-dev** [comment]
I created ..\Debug\Test Spritessheets\flute.wav with a blank file and it compiled.

**Comment by pincoide** [comment]
yeah I'll move that file when I get to update the code again, but first I need to finish the mythic package library upgrade.
